standalone-tools/wss-unified-agent.config

```diff
@@ -139,12 +139,25 @@ resolveAllDependencies=false
 #maven.additionalArguments=
 #maven.projectNameFromDependencyFile=true										 
 
-#gradle.ignoredScopes=
+resolveAllDependencies=false
+archiveExtractionDepth=7
+followSymbolicLinks=true
 gradle.resolveDependencies=true
-gradle.runAssembleCommand=true
-gradle.runPreStep=true
-#gradle.ignoreSourceFiles=true
 gradle.aggregateModules=true
+maven.resolveDependencies=true
+maven.runPreStep=true
+maven.aggregateModules=true
+npm.resolveDependencies=true
+npm.runPreStep=true
+npm.yarnProject=true
+
+
+#gradle.ignoredScopes=
+#gradle.resolveDependencies=true
```

Comment:
Duplicate of line 145

Reply:
Line 156 one is commented , and Line 145 is the actual setting.  Line 156 is commented, as it is a part of the original scan configurations provided by the vendor. The configs are pooled in one place as it makes easier to enable and disable based on need in future.

Reply:
Got it! Thanks